contracts/test/stateverifier.test.ts

```diff
@@ -134,13 +152,20 @@ describe("Verify", function () {
     expect(resultFromStarknetVerifier.toString()).to.be.eq(result[0]);
   });
 
+  it(" test for a poseidon array", async function () {
+    const result = await l1resolverStub.poseidonHashMany(["28355430774503553497671514844211693180464", "0x34894aedf9548524f9e5bb189472d25abe3c38befd577c90886a7c519e5eee4", "0x70c5acad61a421be9c2945b921e263f2699c668f20d31c90660e91b32ea99de"]);
```

Comment:
we could create a constant `FELT_FOR_STARKNET_STATE_V0` for `28355430774503553497671514844211693180464`

Reply:
and for other numbers as well

Reply:
These any be any random values, just checking poseidonHashMany works for any random set of values, I have added a few more tests